### PR TITLE
adding and removing book from the store

### PR DESCRIPTION
**Pull request by EyasuTeshome**
- Configured the Redux Store and import the necessary methods from Redux.
- Write the book's actions and reducer using the ducks pattern.
- In the React component responsible for adding new books set the data inputs in the local React state.
- Once the new book object is ready to be submitted to the Redux store, dispatch a corresponding action.
- In the React component responsible for removing books - implement the Remove event by dispatching a corresponding action.
- Use the redux-logger to check if your application is working correctly.

### CODE REVIEW

File: src/redux/books/books.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  },
+  {
+    id: uuidv4(),
+    bookName: 'Sample Book Two',
+  },
+  {
+    id: uuidv4(),
+    bookName: 'Sample Book Three',
+  },
+];
```
Comment (1GORDON) [inline src/redux/books/books.js]: Great job @EyasuTeshome 🤝

- [ ] As specified in the project requirements, kindly set your initial state to an empty array.

File: src/App.js [hunk 2]
```diff
@@ -3,21 +3,25 @@ import {
...
+        <Navbar />
+        <Routes>
+          <Route path="/" element={<Books />} />
+          <Route path="/categories" element={<Categories />} />
+          <Route path="*" element={<Books />} />
+        </Routes>
+      </div>
+    </Provider>
   );
 }
```
Comment (1GORDON) [inline src/App.js]: Great work @EyasuTeshome 🤝

- [ ] Kindly use ES6 syntax to write all your methods and functions.

**Review by 1GORDON — CHANGES_REQUESTED** [review]
##
## Hi @EyasuTeshome ✋

Good job so far!

## Highlights!
- Followed Gitflow ✔️ 
- Good commit messages ✔️
- Linters are passing ✔️
- Descriptive Pull request ✔️
- Professional readme ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️

- [ ] As specified in the [project requirements](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/add_reducers_an_actions.md), kindly implement the categories actions and reducer.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Reem-lab — APPROVED** [review]
# Hi @EyasuTeshome,👋🏻👋🏻

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉🔥🔥💯💯



## Highlights ✨✨
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Great working on the functionality ✅
📌 Good commit messages ✅
📌 Professional README file. ✅

## Optional suggestions


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**